QUICKSTART.rst

```diff
@@ -136,10 +166,11 @@ What's next?
     changes since the last release              ``CHANGES.md``
 =============================================== =========================
 
-.. _platforms: http://chapel.cray.com/docs/1.13/platforms/index.html
-.. _chapel.cray.com/docs: http://chapel.cray.com/docs/1.13/
-.. _doc/building.rst: http://chapel.cray.com/docs/1.13/usingchapel/building.html
-.. _doc/compiling.rst: http://chapel.cray.com/docs/1.13/usingchapel/compiling.html
-.. _doc/executing.rst: http://chapel.cray.com/docs/1.13/usingchapel/executing.html
-.. _doc/debugging.rst: http://chapel.cray.com/docs/1.13/usingchapel/debugging.html
-.. _doc/bugs.rst: http://chapel.cray.com/docs/1.13/usingchapel/bugs.html
+.. _platforms: http://chapel.cray.com/docs/1.14/platforms/index.html
```

Comment:
Can we actually make these rst cross references with `:ref` rather than hyperlinks?

These were originally links because we used this as our main README on github.

This will avoid having to bump the versions every release.